crs_motion_planning/src/motion_planning_server.cpp

```diff
@@ -293,35 +302,38 @@ class MotionPlanningServer : public rclcpp::Node
 
     for (size_t i = 0; i < request->process_paths.size(); ++i)
     {
+      std::cout << "Planning " << i + 1 << " process of " << request->process_paths.size() << std::endl;
       // Load in current rasters
-      motion_planner_config_->rasters = request->process_paths[i].rasters;
+      motion_planner_config.rasters.clear();
+      motion_planner_config.rasters = request->process_paths[i].rasters;
 
       // Create marker array for original raster visualization
       visualization_msgs::msg::MarkerArray mark_array_msg;
       crs_motion_planning::rasterStripsToMarkerArray(
-          motion_planner_config_->rasters, "world", mark_array_msg, { 1.0, 0.0, 0.0, 1.0 }, -0.01);
+          motion_planner_config.rasters, "world", mark_array_msg, { 1.0, 0.0, 0.0, 1.0 }, -0.01);
```

Comment:
It'd be nice to make the "world" string into a global const variable or preferably use `motion_planner_config.world_frame`

Reply:
Updated